### ISSUE TITLE
In the Preferences in Keybindings in View: Change the shortcut name from "Toggle Messages Window" to "Toggle Message Window" or "Toggle message window"

### ISSUE THREAD

**Issue by ralf3u** [geany/geany#3989]
In the menu in `View` the name is "Message Window" and not "Messages Window".
I think that "Toggle message window" is better than "Toggle Message Window".

**Comment by elextr** [comment]
S/B consistent "View->Toggle Message Window" everywhere.  All entries in the View menu are all capitals, so not "Toggle message window" irrespective of preference.

The keybindings menu has to be changed from "Toggle Messages Window" __and__ the Geany manual entry for that keybinding as well.  PR welcome.

**Comment by techee** [comment]
I think it should be "Show Messages Window" as there's a checkbox next to the entry and it shows whether this "preference" is set. "Toggle" is good for actions like keybindings.

**Comment by elextr** [comment]
Yeah, from that point of view for a tick entry then `Show Message(s) Window` is better.  

So the only issue is message/messages.  Only one of the tabs in the message "Window" (actually its a pane, but lets not get technical :-) is called "Messages", there are also "Status", "Compiler", "Scribble", and "Terminal" as well.  So really calling the menu entry the same as one of the tabs is not the right thing, it is not related to the particular tab, its all tabs.  

Simply removing the `s` is the minimal menu difference unless somebody can think of a better name than "Message Window" or a better name for the "Messages" tab.

(All I could think of was "Miscellaneous Window", but no thats not better).
